admin/broadleaf-open-admin-platform/src/main/java/org/broadleafcommerce/openadmin/server/service/persistence/module/MapStructurePersistenceModule.java

```diff
@@ -142,9 +142,11 @@ public void updateMergedProperties(PersistencePackage persistencePackage, Map<Me
                         MergedPropertyType.MAPSTRUCTUREVALUE
                     );
                 } else {
+                    String valueClassName = mapStructure.getValueClassName();
+                    Class<?>[] entities = persistenceManager.getDynamicEntityDao().getAllPolymorphicEntitiesFromCeiling(Class.forName(valueClassName));
```

Comment:
Curious why this entity array retrieval is written differently than the others

Reply:
It was the first place I changed and forgot to go back and fix it.  I've made the change to match the other locations.